src/Microsoft.AspNetCore.SystemWebAdapters.Authentication/IRemoteAuthenticationService.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Authentication
+{
+    /// <summary>
+    /// A service for authenticating an HTTP request with a remote service.
+    /// </summary>
+    public interface IRemoteAuthenticationService
+    {
+        /// <summary>
+        /// Initializes the remote authentication service for the given scheme.
+        /// </summary>
+        /// <param name="scheme">The scheme whose configuration the remote authentication service should use for authenticating requests.</param>
+        void Initialize(AuthenticationScheme scheme);
```

Comment:
That would work. I chose this approach because it mirrors what `AuthenticationHandler` (which depends on the service) does. Having the initialize here makes it straightforward to have `RemoteAuthenticationAuthHandler`'s initialize method call the service's initialize method.

An alternative would be to have the `RemoteAuthenticationAuthHandler` have a dependency on a service factory, instead, and create the service in its Initialize method.

I don't have a strong preference one way or the other. If you think the factory approach would be cleaner/more maintainable/whatever, let me know and I'll change it.